app/src/components/Modals/add-activity-modal.tsx

```diff
@@ -79,19 +80,6 @@ export type Inputs = {
   subcategoryData: Record<string, SubcategoryData>;
 };
 
-export function determineEmissionsFactorType(factor: EmissionsFactorData) {
-  let sourceName = factor.dataSources
-    ? factor.dataSources[0].datasetName || "Unknown data source"
-    : "Unknown data source";
-  if (sourceName.includes("IPCC") && sourceName.includes("US")) {
-    return "National (US)";
-  } else if (sourceName.includes("IPCC")) {
-    return "IPCC";
-  }
-
-  return sourceName;
-}
-
```

Comment:
I think we might still need this method? Although we are using the names of the data sources now, so maybe that helps...

Reply:
it was duplicated code, we have the exact same logic in determineEmissionsFactorType in ActivityDataTab.tsx